### ISSUE TITLE
A11y_SQLAzureDataStudio_AddaSQLServerBigDataCluster-Controller_Addnewcontroller_AI4D:The name of the edit box is not set according to its label.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9175]
0[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yWCAG2.1;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS2.5.3; 

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Tools:Accessibility insights for desktop
MAS References:MAS2.5.3

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Connections from left menu.
4. Select Add a SQL Server Big Data Clusters.
5. Navigate to + button & hit enter.
6. Launch accessibility insights for desktop and hover over the first edit 

Actual:
On hovering over the first edit box "Cluster management url" the name for it is set as "Cluster management ur please fill out this field".While the label is set as "Cluster management url"

**Same issue in the following page:**
1.Import Bacpac Settings
(issue observed for the 2 edit box present)

**Expected:**
The name should match the label of the edit box.The name should be "Cluster management url" and the words "please fill out this field" could be a placeholder in the edit field.
Screen readers should also announce the name that has been set.

**User Impact:**
Speech to text users will face difficulty in accessing the edit box since the name present on the screen of the label won't match the edit box.

**Attachment for Reference :**
![image](https://user-images.githubusercontent.com/24580570/74708608-ed40fd80-5242-11ea-9732-46e18daea193.png)

**Comment by Charles-Gagnon** [comment]
VS Code took the fix for this so closing this now as we'll get it in the next merge

https://github.com/microsoft/vscode/pull/92531

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_Win32;#A11yMAS;

**Comment by Yashi1201** [comment]
As "SQL Server Big Data Cluster" feature has been removed.
**GithubTags**:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
